bookwyrm/models/session.py

```diff
@@ -0,0 +1,49 @@
+""" functions for managing user sessions """
+import ua_parser
+
+from django.contrib.sessions.backends.db import SessionStore
```

Comment:
Perhaps do the `SessionStore = import_module(settings.SESSION_ENGINE).SessionStore` dance here too, as elsewhere in the code, just to be sure?